deeprank/tools/README.md

```diff
@@ -2,140 +2,6 @@
 
 Here are located all the generic tools used during one or across multiple steps of the workflow. A quick manual is included for the most important tools.
```

Comment:
Not as far as I am aware of, but it could still be useful for a user. I just don't understand why this file provides an explanation of FeatureClass...

Reply:
Maybe it's used in the very early stage of the development. I took a look at it, it's not needed to keep it any more. Could you remove it then?